configuration/appframework/scripts/getDateEnrolledByProgramUuid.js

```diff
@@ -0,0 +1,12 @@
+function getDateEnrolledByProgramUuid(patientPrograms, programUuid) {
+  if (!patientPrograms) {
+    return false;
+  }
+  var i, len = patientPrograms.length;
+  for (i = 0; i < len; ++i) {
```

Comment:
See comment above - use `patientPrograms.forEach`, rather than all of these variables.